packages/cli/src/build-ipfs-connection.util.ts

```diff
@@ -58,6 +47,7 @@ async function createGoIPFS(overrideConfig: Partial<ipfsClient.Options> = {}): P
   const gatewayPort = 9011
   const defaultConfig = {
     ipld: { codecs: [dagJose] },
+    repo: '~/.ipfs',
```

Comment:
Is this the default repo we want to use? I don't mind either way but before it was `.jsipfs` and in `ipfs-daemon` its `.goipfs`

Reply:
true. thanks!